tests/unittests/net/test_networkd.py

```diff
@@ -71,5 +183,89 @@ def test_networkd_render_with_set_name(self):
         assert "ens92" in rendered_content
         assert rendered_content["ens92"] == V2_CONFIG_SET_NAME_RENDERED_ETH1
 
+    def test_networkd_render_dhcp_yes_with_dhcp_overrides(self):
+        with mock.patch("cloudinit.net.get_interfaces_by_mac"):
+            ns = self._parse_network_state_from_config(
+                V2_CONFIG_DHCP_YES_OVERRIDES
+            )
+            renderer = networkd.Renderer()
+            rendered_content = renderer._render_content(ns)
+
+        assert (
+            rendered_content["eth0"] == V2_CONFIG_DHCP_YES_OVERRIDES_RENDERED
+        )
+
+    @pytest.mark.parametrize("dhcp_version", [("4"), ("6")])
+    def test_networkd_render_dhcp_domains_vs_overrides(self, dhcp_version):
+        expectedException = (
```

Comment:
In general, we use snake-case to name vars. Could we rename `expectedException` to `expected_exception`?

Reply:
Whoops! Thanks for catching that mistake.